File: OOPs/Setter and Getter Funtions/Setter_getter.java
public class Setter_getter {
    private String brand;

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getBrand() {
        return brand;
    }

    public static void main(String[] args) {
        Setter_getter myCar = new Setter_getter();
        myCar.setBrand("Toyota");
        System.out.println("Car Brand: " + myCar.getBrand()); 
    }
}

File: OOPs/Setter and Getter Funtions/setter_getter.cpp
#include <bits/stdc++.h>
using namespace std;

class Car {
    private:
        string brand;
    public:
        void setBrand(string brand) {
            this->brand = brand;
        }
        string getBrand() {
            return brand;
        }
};

int main() {
    Car obj;
    obj.setBrand("Audi");
    cout << obj.getBrand() << endl;
    return 0;
}